src/polybench/POLYBENCH_2MM-OMP.cpp

```diff
@@ -53,26 +53,25 @@ void POLYBENCH_2MM::runOpenMPVariant(VariantID vid)
 
   POLYBENCH_2MM_VIEWS_RAJA;
 
-  auto poly_2mm_lam1 = [=](Index_type /*i*/, Index_type /*j*/, Index_type /*k*/,                           Real_type &dot) {
+  auto poly_2mm_lam1 = [=](Real_type &dot) {
```

Comment:
@vsrana01 have you checked whether these changes do not adversely affect performance across a range of compilers?  

I don't think we want to make changes like this in this PR, since they are orthogonal. We can think about adding additional variants like this later to further stress compilers.

Reply:
@rhornung67 These changes may be needed with the latest Lambda changes:  if not all segments are in active loops, the Lambda<N> form will static_assert out